package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "group-capstone",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "main": "jest.config.js",
+  "scripts": {
+    "test": "jest",
+    "build": "webpack --mode production",
+    "start": "webpack --mode development --config webpack.config.js --watch"
+  },
```

Comment:
- [ ] Please update your Package.json configuration to run and start the app in the browser instead of using the index.html inside the dist folder. ✍🏻 Actually I can't start the local server (npm start or npm run dev, ....)